.github/workflows/check-release.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     strategy:
       matrix:
         group: [check_release, link_check]
+    env:
+      PIP_ONLY_BINARY: "pyrsistent"
```

Comment:
It will most certainly not fix the releaser execution for making releases.

But I'm not sure it will be a _good_ fix. Should we support defining environment variables as `jupyter-releaser` options?

Reply:
Maybe an `export PIP_ONLY_BINARY=pyrsistent` in a `before-prep-git` hook might do the trick?

Reply:
In 6ee4ab8, I try the hook approach but it does not work.

Reply:
ah it's possible that env var is not persisted once the `before-prep-git` hook is finished.